bigtable/quickstart_happybase/main.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# [START bigtable_quickstart_happybase]
+import argparse
+import json
+
+from google.cloud import bigtable
+from google.cloud import happybase
+
+
+def main(project_id, instance_id, table_id):
```

Comment:
Ditto on showing default values here so it's clear right away the types and what the values should look like.

Reply:
Done in 56393fe